nvdaHelper/remote/textFromIAccessible.cpp

```diff
@@ -29,45 +69,47 @@ bool getTextFromIAccessible(
 	bool recurse,
 	bool includeTopLevelText
 ) {
-	bool gotText = false;
-	IAccessibleText* paccText = NULL;
-	if (pacc2->QueryInterface(IID_IAccessibleText, (void**)&paccText) != S_OK) {
-		paccText = NULL;
+	if (!pacc2) {
+		return false;
 	}
+
+	bool gotText = false;
+	CComQIPtr<IAccessibleText, &IID_IAccessibleText> paccText(pacc2);

```

Comment:
I think it is possible to not have to specify the IID on CComQIPtr, as the template should calculate it from _uuidof(interface).

Reply:
Yes, I believe so. I chose to make it explicit because I wasn't sure of any 'gotchas' involved. If your confident about this I'll remove the extra param. In some places we already rely on the calculated UUID.

Reply:
I can't see any differences in the logic.

Reply:
This has been addressed.